Latex/inhalt/Kapitel1.tex

```diff
@@ -0,0 +1,12 @@
+%Alle Seiten beginnen mit der Oberüberschrift
+\section{Einleitung}
+Die ist eine PDF mit der Vorlage erstellt, um die Funktion und Formatierung dieser zu zeigen.
+
+Diese Vorlage\onlinezitat{SCZYRBA2020} ist ein Gemeinschaftsprojekt im Rahmen unseres Studiums.
+Der Docker-Container\onlinezitat{HILLE2021} gehört dazu.
+
+Weitere Hinweise befinden sich in der Readme.md oder im Wiki.
+Eine ausführliche Dokumentation zu diesem Dokument wird folgen.
+
+\section{Kapitel 1}
```

Comment:
Ich würde die Beispiele aus den anderen Kapiteln nicht ersatzlos streichen, sondern hier eins oder mehrere Kapitel mit diesen erstellen. Wie man ein Bild nutzt o.Ä. ist trotzdem gut zu wissen.

Reply:
Dafür wird es ein ordentliches Dokument mit Beispielen in näherer Zukunft gebe...
Daher Beleg-Branch beachten...

Reply:
ich würde nicht die anderen Bestandteile löschen bevor das da ist. gern im Zuge der Änderungen in Beleg-Branch. aber solange das nicht drin ist, sollte das andere drin bleiben. Ist ja nur copy paste..

Reply:
Ist doch nur Dev... Dev ist nicht Master und genau sowas soll in Dev :D